lib/bundler.rb

```diff
@@ -195,6 +195,16 @@ def tmp(name = Process.pid.to_s)
 
     def rm_rf(path)
       FileUtils.remove_entry_secure(path) if path && File.exist?(path)
+    rescue ArgumentError => e
+      if /world writeable/ =~ e.message
```

Comment:
agree, but I'd rather we check for the error condition ourselves rather than checking for a regex match on the thrown exceptions message